README.md

```diff
@@ -3,6 +3,16 @@
 Some scripts I've written for [Merlin's UdonSharp](https://github.com/Merlin-san/UdonSharp)
 
 
+#### > Boost.cs
+
+Player within range of the `referencePoint` gameobject will be thrown into direction of `targetPoint` with an assigned speed. 
+
+
+#### > PlayerMods.cs
+
+A script for altering player settings such as speed and jump. Allows to use old VRChat locomotion behaviour.
```

Comment:
Replace second sentence with:
> Optionally allows the use of the old VRChat locomotion behaviour.

This could include more information differentiating between the new and old locomotion behaviours too.

Reply:
Fixed with a new commit